pmoves/tools/secrets_funnel_populate.py

```
@@ -0,0 +1,845 @@
+        os.chmod(tmp_path, 0o600)
+
+        # Write key=value to temp file
+        with os.fdopen(fd, "w") as f:
+            f.write(f"{name}={value}\n")
+        fd = None  # fdopen closed it
+
+        # Build CHIT CLI command -- NEVER pass value as CLI arg
+        chit_cli = PMOVES_ROOT / "tools" / "chit_cli.py"
+        if chit_cli.exists():
+            # Use CHIT CLI with --from-file
+            result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)